src/main/webapp/profile.js

```diff
@@ -13,30 +13,52 @@
 // limitations under the License.
 
 function onProfileLoad() {
+  supplyLogoutLink();  
+  autofillForm(getUserIfRegistered());  
   prepareFormValidation();
 }
 
-// Allows certain fields in the profile to be edited, hides edit button, and displays update button. 
+// Returns a Person if they registered in the past. If not, redirect to  
+// registration page.
+function getUserIfRegistered(){
+  fetch('/login')
+  .then(response => response.json())
+  .then(status => fetch(`/person?email=${status.email}`))
+  .then(response => {
+    if (response.redirected) {
```

Comment:
Indent this by two.

Reply:
Why? The if is indented.

Reply:
(because the `.then` needs to be indented more)

Reply:
Gotcha, done.